.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npm run lint:fix
+# npm run prettier:fix
```

Comment:
Is this meant to be commented out?

Reply:
No, It's just so I don't commit the formatted files yet.